docs/examples/feature_embedding.ipynb

```diff
@@ -0,0 +1,432 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open In\n",
+    "Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alibaba/feathub/blob/master/docs/examples/feature_embedding.ipynb)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feature Embedding\n",
+    "\n",
+    "Feature embedding is a way to translate a high-dimensional feature vector to a\n",
+    "lower-dimensional vector, where the embedding can be learned and reused across\n",
+    "models. In this example, we show how one can define feature embeddings via\n",
+    "Python UDF (User Defined Function).\n",
+    "\n",
+    "We use a sample hotel review dataset downloaded from [Azure-Samples\n",
+    "repository](https://github.com/Azure-Samples/azure-search-sample-data). The\n",
+    "original dataset can be found\n",
+    "[here](https://www.kaggle.com/datasets/datafiniti/hotel-reviews).\n",
+    "\n",
+    "For the embedding, a pre-trained [HuggingFace Transformer\n",
+    "model](https://huggingface.co/sentence-transformers) is used to encode texts\n",
+    "into numerical values. The text embeddings can be used for many NLP problems\n",
+    "such as detecting fake reviews, sentiment analysis, and finding similar hotels,\n",
+    "but building such models is out of scope and thus we don't cover that in this\n",
+    "notebook."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install dependencies\n",
```

Comment:
Would it be better to uncomment these code so that users can install the dependencies easily by just clicking the cell?